### DISCUSSION TITLE
Welcome to WebRTC.rs Discussions!

### DISCUSSION THREAD

**Discussion by rainliu** [webrtc-rs/webrtc#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dbidwell94** [comment]
Hello there! Just wanted to say I'm pretty excited for this implementation! 

I've been looking for a browerless way to implement webrtc for a couple weeks now. I'd like to create a user owned media streaming server. I tried once before in .net core but it would have required each user who has a server to also put SSL certs on their servers to prevent browser mixed-content errors.

WebRTC, however, handles encryption behind the scenes no there isn't an SSL cert I would have to put on users' servers :grinning:
